scripts/qa_crowbarsetup.sh

```diff
@@ -2531,7 +2531,7 @@ function adapt_dns_for_docker()
 function oncontroller_testsetup()
 {
     . .openrc
-
+    [[ $want_mtu_size ]] && safely ping -M do -c 1 -s $(( want_mtu_size - 28 )) $adminip
```

Comment:
- Can you add a comment about the 28?
- What about testing `storage` and `os_sdn` net?